src/Illuminate/Support/Str.php

```diff
@@ -1112,6 +1113,61 @@ public static function createUuidsUsing(callable $factory = null)
         static::$uuidFactory = $factory;
     }
 
+    /**
+     * Set the sequence that will be used to generate UUIDs.
+     *
+     * @param  array  $sequence
+     * @param  callable|null  $whenMissing
```

Comment:
Any chance to change the `callable` type hint to something static analysis better understands? The recent updates to the Collections doc blocks were a really nice QoL improvement and I'd love for more code to be annotated like that :)

```php
/**
 * @param null|callable(): UuidInterface $whenMissing
 */
```

Same for all the other methods that take a callback as a parameter.